pom.xml

```diff
@@ -160,7 +160,7 @@
         <dependency>
             <groupId>com.cloudbees</groupId>
             <artifactId>groovy-cps</artifactId>
-            <version>1.6</version>
+            <version>1.8-SNAPSHOT</version>
```

Comment:
Unfortunately this pulls in the broken changes in 1.7 reported in JENKINS-34064 which https://github.com/cloudbees/groovy-cps/pull/24/commits/ae43475fe6a814e2562978fcdd03e30a2d8d91b1 tries to deal with.